.github/workflows/portal-pipeline.yml

```diff
@@ -118,7 +118,8 @@ jobs:
           CYPRESS_BASE_URL: ${{ env.URL }}
           CYPRESS_KUBE_API_TOKEN: ${{ env.KUBE_API_TOKEN }}
           CYPRESS_KUBE_API_SERVER: ${{ env.KUBE_API_SERVER}}
-
+          CYPRESS_AGENT_NAMESPACE: ${{ env.CS_MODE_NAMESPACE}}
```

Comment:
We will have to make both lowercase `cypress _agent_namespace` & `cypress_agent` or In the tests, the envs should be taken as `AGENT_NAMESPACE` & `AGENT`